src/modules/IscsiLioServer.rb

```diff
@@ -169,6 +169,18 @@ def getServiceStatus
         @statusOnStart = true
         @serviceStatus = true
       end
+      status = File.exist?("/etc/target/lio_setup.sh")
+      if !status
+      SCR.Execute(path(".target.bash_output"), "touch /etc/target/lio_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "chmod 755 /etc/target/lio_setup.sh")
+      SCR.Execute(path(".target.bash_output"), "echo '#'!/bin/bash > /etc/target/lio_setup.sh")
+      end
+      status = File.exist?("/etc/target/tcm_setup.sh")
+      if !status
+      SCR.Execute(path(".target.bash_output"), "touch /etc/target/tcm_setup.sh")
```

Comment:
And please follow the indentation if two spaces. In fact, if you put it into separate (private) method, you can do it this way:

    return true if /file exists/
    otherwise do something